server/server.py

```diff
@@ -1,25 +1,17 @@
 # server.py
-from flask import Flask, render_template, request
+from flask import Flask, render_template, request, url_for
 import speak
 
-# text = "AI AI minha machadinha !!"
-# lang = "pt"
-
 app = Flask(__name__, static_folder="../static/dist", template_folder="../static")
 @app.route("/", methods=["GET","POST"])
 def index():
     return render_template("index.html")
 
-@app.route("/hello", methods=["GET","POST"])
-def hello():
-    text=''
-    lang=''
+@app.route("/hello/<text>/<lang>")
+def hello(text,lang):
     if request.method == "GET":
-        text = request.form["text"]
-        lang = request.form["lang"]
-        print(text)
         speak.get_speak(text,lang)
-        return flask.url_for ("hello", filename="mp3_path")
+        return flask.url_for("hello", filename="mp3_path")
 
 if __name__ == "__main__":
     app.run()
```

Comment:
- A nitpick Make sure to add a newline after this one too

Reply:
Done !!